viewlet/tests/test_viewlet.py

```diff
@@ -240,6 +238,7 @@ def test_jinja_tag(self):
         html = template.render({"extra": "Räksmörgås", "viewlet_arg": "wörld"})
         self.assertEqual(html.strip(), "<h1>RäksmörgåsHello wörld!%s</h1>" % self.tail)
 
+    # TODO: port this test
     @skipIf(six.PY3, "TODO: coffin fails for Python 3.x?")
     @skipIf(django.VERSION > (1, 7), "Coffin does not support django > 1.7?")
     def test_custom_jinja2_environment(self):
```

Comment:
This test was already disabled before for the latest django version, but I'd like to at least try to make it work

Reply:
Jinja2 with Django has changed a lot since this library was written. I think `django-jinja` is the preferred lib. Not jingo or coffin anymore. So we should probably use that lib instead.